packages/contracts-bedrock/test/L2/L2ToL2CrossDomainMessenger.t.sol

```diff
@@ -248,6 +299,48 @@ contract L2ToL2CrossDomainMessengerTest is Test {
         );
     }
 
+    /// @dev Tests that the `relayMessage` function reverts if called by an address other than the specified entrypoint.
+    function testFuzz_relayMessage_wrongEntrypoint_reverts(
```

Comment:
This test should respect the code order and go after the `testFuzz_relayMessage_targetL2ToL2CrossDomainMessenger_reverts` one